api/app/tests/apps/users/v1/test_permissions.py

```diff
@@ -0,0 +1,89 @@
+from django.contrib.auth.models import Permission
+
+from tests.test import SIAReadUserMixin, SignalsBaseApiTestCase
+
+
+class TestRolesViews(SIAReadUserMixin, SignalsBaseApiTestCase):
+    def test_get_roles(self):
+        permission_count = Permission.objects.count()
+
+        self.client.force_authenticate(user=self.sia_read_user)
+
+        response = self.client.get('/signals/v1/private/permissions/')
```

Comment:
Consider putting: '/signals/v1/private/permissions/' as a property in the `__init__` of the `TestCase`.